src/me/mrCookieSlime/Slimefun/listeners/BowListener.java

```diff
@@ -95,23 +107,8 @@ public void run() {
 
 	@EventHandler
 	public void onArrowSuccessfulHit(EntityDamageByEntityEvent e) {
-		if (!(e.getDamager() instanceof Arrow)) return;
-
-		handleGrapplingHook((Arrow) e.getDamager());
-		
-		if (Variables.arrows.containsKey(e.getDamager().getUniqueId()) && e.getEntity() instanceof LivingEntity) {
-			for (ItemHandler handler: SlimefunItem.getHandlers("BowShootHandler")) {
-				if (((BowShootHandler) handler).onHit(e, (LivingEntity) e.getEntity())) break;
-			}
-			Variables.arrows.remove(e.getDamager().getUniqueId());
-		}
-		
-		if (Variables.arrows.containsKey(e.getDamager().getUniqueId()) && e.getEntity() instanceof LivingEntity) {
-			for (ItemHandler handler: SlimefunItem.getHandlers("BowShootHandler")) {
-				if (((BowShootHandler) handler).onHit(e, (LivingEntity) e.getEntity())) break;
-			}
-			Variables.arrows.remove(e.getDamager().getUniqueId());
-		}
```

Comment:
The above part with the BowShootHandler has to stay there or you are basically removing all Bows from Slimefun and its Addons.

Why its actually twice there is unclear to me though :?